src/main/java/com/artipie/asto/ext/KeyLastPart.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.asto.ext;
+
+import com.artipie.asto.Key;
+
+/**
+ * Last part of the storage {@link com.artipie.asto.Key}.
+ * @since 0.24
+ */
+public final class KeyLastPart {
+
+    /**
+     * Origin key.
+     */
+    private final Key origin;
+
+    /**
+     * Ctor.
+     * @param origin Key
+     */
+    public KeyLastPart(final Key origin) {
+        this.origin = origin;
+    }
+
+    /**
+     * Get last part of the key.
+     * @return Key last part as string
+     */
+    public String get() {
+        final String[] parts = this.origin.string().split("/");
```

Comment:
@olenagerasimova I think we need to remove trailing `/` chars before to avoid empty last part, e.g. `/foo/bar/` an `/foo/bar` should be equal for this class